app/controllers/todo_list_controller.rb

```diff
@@ -1,7 +1,8 @@
 class TodoListController < ApplicationController
     def index
-        @todo_list = TodoList.where(id_usuario: current_user.id)
-
+        @todo_list_done_homework = TodoList.where(id_usuario: current_user.id, done_homework: true)
+        @todo_list_pending_homework = TodoList.where(id_usuario: current_user.id, done_homework: false)
+        
         case params[:format] 
```

Comment:
There is no need for the case here if inside of `respond_to do |format|` you put `format.pdf do ... end` you get the same result.